dolphinscheduler-ui/src/views/resource/file/edit/index.tsx

```diff
@@ -109,7 +109,21 @@ export default defineComponent({
           </div>
         ) : (
           <NSpace justify='center'>
-            <NSpin show={true} />
+            <NSpace vertical>
```

Comment:
It seems that it don't need to wrap a `NSpace` by another one.

Reply:
Yes, you are right!

But, I think adding the NSpace tag is to make the UI look more beautiful!

Commenting out the span tag：

![reply1](https://user-images.githubusercontent.com/20340304/211232714-d3489c7b-43c7-46a3-80bc-b04b0d00703c.jpg)

After
![reply2](https://user-images.githubusercontent.com/20340304/211232746-7b541a71-7234-4d17-85cb-0e195215cdfb.jpg)